packages/datagateway-search/src/table/datafileSearchTable.component.tsx

```diff
@@ -201,6 +340,10 @@ const mapDispatchToProps = (
               id: { in: luceneData },
             }),
           },
+          {
+            filterType: 'include',
+            filterValue: '"dataset"',
```

Comment:
Again, we need a more detailed include i.e.: `filterValue: JSON.stringify({dataset: {investigation: { investigationInstruments: 'instrument' }}})`

Reply:
Very handy for improving my understanding of how these queries are structured!